mammooc-course-evaluation-form.html

```diff
@@ -58,41 +64,43 @@
+                        <img id="profile_image"></img>
+                        <div id="profile_name"></div>
+                    </template>
+                    <div class="description">
+                        <paper-textarea placeholder="{{localize('descriptionplaceholder')}}" name="description" label="{{localize('descriptionlabel')}}" id="description"></paper-textarea>
+                    </div>
                 </div>
-            </div>
-            <label id="statusLabel">
-                {{localize("status")}}
-                </label>
-            <paper-radio-group id="status" aria-labelledby="statusLabel" selected="enrolled">
-                <paper-radio-button name="aborted">
-                    {{localize("aborted")}}
-                </paper-radio-button>
-                <paper-radio-button name="enrolled">
-                    {{localize("enrolled")}}
-                </paper-radio-button>
-                <paper-radio-button name="finished">
-                    {{localize("finished")}}
-                </paper-radio-button>
-            </paper-radio-group>
-            <paper-checkbox id="anonymously" name="anonymously">
-                {{localize("submitanonymously")}}
-            </paper-checkbox>
-            <flexible-rating id="stars" name="stars" required></flexible-rating>
-            <paper-button class="submit-button" raised on-click="_submitForm">
-                {{localize("submit")}}
-            </paper-button>
-        </form>
+                <label id="statusLabel">
+                    {{localize("status")}}
+                    </label>
+                <paper-radio-group id="status" aria-labelledby="statusLabel" selected="enrolled">
+                    <paper-radio-button name="aborted">
+                        {{localize("aborted")}}
+                    </paper-radio-button>
+                    <paper-radio-button name="enrolled">
+                        {{localize("enrolled")}}
+                    </paper-radio-button>
+                    <paper-radio-button name="finished">
+                        {{localize("finished")}}
+                    </paper-radio-button>
+                </paper-radio-group>
+                <paper-checkbox id="anonymously" name="anonymously">
+                    {{localize("submitanonymously")}}
+                </paper-checkbox>
+                <flexible-rating id="stars" name="stars" required></flexible-rating>
+                <paper-button class="submit-button" raised on-click="_submitForm">
+                    {{localize("submit")}}
+                </paper-button>
+            </form>
+        </iron-form>
 
         <!-- Send form if user is not logged in -->
-        <form method="post" target="_blank" action="https://mammooc.org/evaluations/login_and_save" id="nativeForm">
+        <form method="post" target="_blank" action="https://mammooc-dev.herokuapp.com/evaluations/login_and_save" id="nativeForm">
```

Comment:
Is this change on purpose?

Reply:
Yes, it is. Due to a mistake, the dev branch currently holds this "wrong" URL, which is only applicable to our master. I will double check the URLs prior to publishing a new release.